### PR TITLE
SQS Increment service. Дмитрий Абрамов и Юрий Парадовский.

### PR DESCRIPTION
**Pull request by karvozavr**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+group 'ru.spb.hse.karvozavr'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    compile group: 'com.amazonaws', name: 'aws-java-sdk', version: '1.11.564'
```
Comment (TanVD) [inline app/build.gradle]: Do not use whole AWS SDK - use only needed services

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.3.30'
```
Comment (TanVD) [inline app/build.gradle]: Why not 31?

File: app/src/main/kotlin/ru/spb/hse/karvozavr/increment/QueueManager.kt [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    private val sqs: AmazonSQS = AmazonSQSClientBuilder.standard()
+        .withEndpointConfiguration(
+            AwsClientBuilder.EndpointConfiguration(
+                "http://localstack:4576",
+                "eu-central-1"
+            )
+        )
+        .build()
+
+    public fun checkQueueExists(queueName: String): Boolean {
```
Comment (TanVD) [inline app/src/main/kotlin/ru/spb/hse/karvozavr/increment/QueueManager.kt]: Public is not needed in Kotlin

File: app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt [hunk 4]
```diff
@@ -0,0 +1,43 @@
+package ru.spb.hse.karvozavr.increment
+
+import com.amazonaws.services.sqs.model.Message
+
+class IncrementService(private val queueA: String, private val queueB: String) {
+
+    private val queueAUrl: String
+    private val queueBUrl: String
+    private val sqsManager = QueueManager()
```
Comment (TanVD) [inline app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt]: Use `object` for such purposes

**Review by TanVD — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 2 above)
Comment (karvozavr) [reply app/build.gradle]: Я не обновился вовремя и так сгенерилось.

File: app/src/main/kotlin/ru/spb/hse/karvozavr/increment/QueueManager.kt (hunk 3 above)
Comment (karvozavr) [reply app/src/main/kotlin/ru/spb/hse/karvozavr/increment/QueueManager.kt]: Fixed.

File: app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt (hunk 4 above)
Comment (karvozavr) [reply app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt]: @TanVD, please, could you be more clear about purposes and the use case of `object` you meant?

File: app/build.gradle (hunk 2 above)
Comment (karvozavr) [reply app/build.gradle]: Fixed

File: app/build.gradle (hunk 1 above)
Comment (karvozavr) [reply app/build.gradle]: Fixed.

File: app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt (hunk 4 above)
Comment (TanVD) [reply app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt]: You are using QueueManager as a singleton. Take a look at the `object` construct in Kotlin - it is `out of the box` singleton.

Comment (subject-name-here) [reply app/src/main/kotlin/ru/spb/hse/karvozavr/increment/IncrementService.kt]: fixed

**Comment by subject-name-here** [comment]
@TanVD !

**Pull request merged**